File: GTTS/INL.TrainingService.Models/GetTrainingEventVisaCheckLists_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public class GetTrainingEventVisaCheckLists_Item : IGetTrainingEventVisaCheckLists_Item
    {
        public long PersonID { get; set; }
        public long? TrainingEventID { get; set; }
        public long? TrainingEventVisaCheckListID { get; set; }
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        public string AgencyName { get; set; }
        public string VettingStatus { get; set; }
        public bool? HasHostNationCorrespondence { get; set; }
        public bool? HasUSGCorrespondence { get; set; }
        public bool? IsApplicationComplete { get; set; }
        public DateTime? ApplicationSubmittedDate { get; set; }
        public bool? HasPassportAndPhotos { get; set; }
        public string VisaStatus { get; set; }
        public string TrackingNumber { get; set; }
        public string Comments { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-visa-check-lists_item.ts


import { IGetTrainingEventVisaCheckLists_Item } from './iget-training-event-visa-check-lists_item';

export class GetTrainingEventVisaCheckLists_Item implements IGetTrainingEventVisaCheckLists_Item {
  
	public PersonID: number = 0;
	public TrainingEventID?: number;
	public TrainingEventVisaCheckListID?: number;
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public AgencyName: string = "";
	public VettingStatus: string = "";
	public HasHostNationCorrespondence?: boolean;
	public HasUSGCorrespondence?: boolean;
	public IsApplicationComplete?: boolean;
	public ApplicationSubmittedDate?: Date;
	public HasPassportAndPhotos?: boolean;
	public VisaStatus: string = "";
	public TrackingNumber: string = "";
	public Comments: string = "";
  
}






